rust/azure_iot_operations_protocol/src/rpc_command/executor.rs

```diff
@@ -212,6 +212,42 @@ impl<TResp: PayloadSerialize> ResponseBuilder<TResp> {
         }
         Ok(())
     }
+
+    /// Helper function to add user-defined application error headers to `response`.
```

Comment:
You could link to the `Response` struct here, I believe it is [`Response`]

```suggestion
    /// Helper function to add user-defined application error headers to [`Response`].
```